src/colorizer/ColorizeCanvas2D.ts

```diff
@@ -362,7 +368,7 @@ export default class ColorizeCanvas2D implements IRenderCanvas {
   }
 
   private setOutlineColor(color: Color): void {
-    this.setUniform("outlineColor", color);
+    this.setUniform("outlineColor", color.clone().convertLinearToSRGB());
```

Comment:
Fixes a bug where certain colors would look incorrect in the 2D and 3D viewports.